docs/reference/filters.md

```diff
@@ -1511,7 +1511,7 @@ It is also possible to enable this behavior centrally for a Skipper instance wit
 the -rfc-patch-path flag. See
 [URI standards interpretation](../../operation/operation/#uri-standards-interpretation).
 
-## bearerinjector
+## bearerInjector
```

Comment:
this was actually cased like that, not saying that it's the right way ;)
https://github.com/zalando/skipper/blob/master/filters/auth/bearer.go#L9

Reply:
In that case, I will revert this change from current PR as its not relevant. I will create an issue to discuss if its possible to change the name by keeping it backward compatible or would it be a breaking change